src/definitions/behaviors/api.js

```diff
@@ -148,8 +148,8 @@ $.api = $.fn.api = function(parameters) {
               module.error(error.noStorage);
               return;
             }
-            response = sessionStorage.getItem(url);
-            module.debug('Using cached response', url, response);
+            response = sessionStorage.getItem(url + (typeof settings.data === "string" ? settings.data : JSON.stringify(settings.data,Object.keys(settings.data).sort())));
```

Comment:
I suggest to extract function, since these duplications are non-trivial.

```javascript
function normalize(data) {
  return (typeof data === "string" ? data : JSON.stringify(data, Object.keys(data).sort())
}

// use-site
normalize(settings.data)
```

Reply:
Done by b5aa6fca667e03759d1f1eb841f2a6a78ad05a02 
I decided to do it for the dedicated settings.data variable and put it under the `get` bahavior. This way it is also possible to retrieve this separately via behavior if needed